packages/react-devtools-shared/src/backend/types.js

```diff
@@ -75,7 +75,7 @@ export type WorkTagMap = {
   Throw: WorkTag,
 };
 
-export type NativeType = Object;
+export type HostInstance = Object;
```

Comment:
I find it a bit confusing, because Host is essentially React DevTools backend, and this is HostInstance, which is not about React DevTools backend, but element instance.

Maybe something like `InstanceForHost` or `ElementInstanceForHost`?

Reply:
Well I think of Host is the environment that the backend is running inside. Not the backend itself.

It's also what we call this elsewhere in Fiber. It gets shortened into the name `Instance` inside the "Host Config" but the long name we refer it by is Host Instance. We could potentially rename it to `HostInstance` even in the Configs and then it would have the same name everywhere.

This name already shows up even in the DevTools interface too e.g. through the injected `findFiberByHostInstance`.

Also, not Elements in the React DevTools refer to anything that has a Host Instance. Only a subset of Elements are represented by a Host Instance.

I have a follow up draft PR where I'm introducing an Instance to back stuff for the DevTools specifically which I'm calling a `DevtoolsInstance` which is more 1:1 with the Elements tab instances.

Although I think we should probably rename Element to Component in the Elements tab because in React an Element is what you return from JSX but a Component instance is the stateful thing you get as a result of rendering an Element.

Reply:
Unrelated but ReactComponentInfo started out as an idea of being more like an instance but it's not actually a stateful thing and I wonder if it shouldn't just be modeled as a ReactElement. It already has mostly the same fields. It should have props too. ReactElement is missing `env` but technically it should have it: https://github.com/facebook/react/blob/ec98d36c3a47e9f02c45cc14ff6046fff8c27458/packages/react-client/src/ReactFlightClient.js#L699-L703